org.metaborg.spoofax.core/src/main/java/org/metaborg/spoofax/core/stratego/StrategoCommon.java

```diff
@@ -82,7 +84,7 @@
         String strategy) throws MetaborgException {
         List<MetaborgException> exceptions = Lists.newArrayList();
         for(ILanguageComponent component : impl.components()) {
-            if(component.facet(StrategoRuntimeFacet.class) == null) {
+            if(component.facet(DynamicClassLoadingFacet.class) == null) {
```

Comment:
I think this method should call `invoke(ILanguageComponent, FileObject, IStrategoTerm,
String)`, which does not exist yet.

Reply:
Same issue as the previous